### DISCUSSION TITLE
Cant commit a file in branch (Skills-introduction to Github)

### DISCUSSION THREAD

**Discussion by AnsarMahir** [skills/.github#250] [Q&A]
I'm following skills-introduction-to-GitHub. As the second step on the tutorial, I have to create a "PROFILE.md" File in the branch created. Whenever I try to create the file and click on commit changes, Then I get a message saying "Error committing your change : File could not be edited". I tried several times and got the same message. 

After a while, I compared the main branch with the branch i tried to create the new file. The new file is there in the branch. I also compared the two branches and saw that the new file is there. But, the steps are still not moving forward, What can i do ??

**Comment by sinsukehlab** [comment]
It's not your fault. There was an incident again.
See: https://www.githubstatus.com/incidents/17c40y7spfgl
